src/Evolution/DiscontinuousGalerkin/SlopeLimiters/Minmod.cpp

```diff
@@ -95,10 +95,35 @@ bool limit_one_tensor(
       const bool has_neighbors = (externals.find(dir) == externals.end());
       if (has_neighbors) {
         const auto& neighbor_ids = element.neighbors().at(dir).ids();
-        ASSERT(neighbor_ids.size() == 1,
-               "Minmod does not yet support h-refinment");
-        // We've checked for a single neighbor, so just read first element:
-        const auto dir_key = std::make_pair(dir, *(neighbor_ids.cbegin()));
+
+        // Average neighbor_size toward dim, side:
```

Comment:
average neighbor_size normal to face

Reply:
Yes: only the component of `neighbor_size` that is normal to the face is needed, so I only do the work of computing the normal component.